qiskit/result/result.py

```diff
@@ -231,20 +231,20 @@ def get_memory(self, experiment=None):
         except KeyError:
             raise QiskitError('No memory for experiment "{}".'.format(experiment))
 
-    def get_counts(self, experiment=None):
+    def get_counts(self, experiment=None, include_zeros=False):
         """Get the histogram data of an experiment.
 
         Args:
             experiment (str or QuantumCircuit or Schedule or int or None): the index of the
                 experiment, as specified by ``get_data()``.
+            include_zeros (bool): If `True`, it includes the counts in zero. Default `False`.
```

Comment:
Same comment here as in the class docstring, this needs a better description of how it works.

Reply:
I think I improved the description in 3f74ff3b4